pkg/kubernetes/scripts/virtual-machines/components/createVmDialog.jsx

```diff
@@ -98,16 +169,57 @@ class CreateVmDialog extends React.Component {
                 <label className="help-block" id='resource-text-error'>{this.state.resourceError}</label>);
         }
 
+        let name = this.state.filename ? this.state.filename : _("No files found");
```

Comment:
Nitpick (not important): for this case the easier and more usual pattern is `a || b` instead of `a ? a : b`.